src/main/java/seedu/address/logic/commands/AddLabCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LAB;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.lab.Lab;
+
+/**
+ * Adds a person to the address book.
+ */
+public class AddLabCommand extends Command {
+
+    public static final String COMMAND_WORD = "labadd";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a lab to all students in the TAddressBook. "
+            + "Parameters: "
+            + PREFIX_LAB + "LAB NUMBER "
```

Comment:
Maybe should put like `LAB_NUMBER` to make it clearer that it's only one entity

Reply:
Good suggestion, I will change it.